src/VS.Web.CG.Mvc/Templates/ControllerGenerator/ApiControllerWithContext.cshtml

```diff
@@ -89,7 +89,7 @@ namespace @Model.ControllerNamespace
 
         // POST: @routePrefix
         // To protect from overposting attacks, please enable the specific properties you want to bind to, for
-        // more details see https://aka.ms/RazorPagesCRUD.
+        // more details see https://go.microsoft.com/fwlink/?linkid=2123754.
```

Comment:
It won't let me suggest but needs a comma, as the one above

// more details, see https://go.microsoft.com/fwlink/?linkid=2123754.